students/wooseok-choi/user/views.py

```diff
@@ -1 +1,30 @@
+import json
 
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models import User
+from user.validate import *
```

Comment:
```suggestion
from user.validate import 
```
실제 코드에서 *을 사용하는 것은 좋지 못한 습관입니다. 사용할 모듈 또는 함수를 명확히 하는 것이 좋습니다.